### PR TITLE
Refactor Buttons Enabled/Disabled State Styling

### PR DESCRIPTION
**Pull request by pavelprystinka**
## Purpose

* Refactor Buttons Enabled/Disabled State Styling
* Fixes disabled buttons not legible in a dark mode

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code run it and observe buttons in light and dark modes

### CODE REVIEW

File: AzureCalling/app/src/main/res/color-night/button_filled.xml
Comment (iaulakh) [inline AzureCalling/app/src/main/res/color-night/button_filled.xml:1]: We already have button_filled.xml in drawable. would it be possible to create only only button_filled.xml in our code and use color from colors.xml & colors.xml(night)?
For example:

button_filled.xml
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:color="@color/color1" android:state_enabled="true" />
    <item android:color="@color/color2" android:state_enabled="false" />
</selector>

colors.xml (night)
<resources>
    <color name="color2">@android:color/holo_orange_dark</color>
    <color name="color1">@android:color/holo_blue_bright</color>
</resources>

colors.xml
<color name="color2">@android:color/holo_red_dark</color>
<color name="color1">@android:color/holo_green_light</color>

Dark Mode:
Screen Shot 2021-04-09 at 10 32 32 PM

Light Mode:
Screen Shot 2021-04-09 at 10 33 22 PM

We can follow above for others controls.

Comment (pavelprystinka) [reply AzureCalling/app/src/main/res/color-night/button_filled.xml:1]: Discussed offline:
We will re-use already defined accentColor but will not define new singe use dedicated for button color.

File: AzureCalling/app/src/main/assets/appSettings.properties
Comment (iaulakh) [inline AzureCalling/app/src/main/assets/appSettings.properties:2]: Please undo this file changes.

Comment (pavelprystinka) [reply AzureCalling/app/src/main/assets/appSettings.properties]: Done.

**Review by iaulakh — APPROVED** [review]
looks Good Pavel.

Please wait for other team member reviews to see if we want to define all color in one file and want to have selectors file independent of color. 

Once, this is merged Please also send PR for teams_interop_merge as we added additional radio box and texts for join call screen.

**Review by waynemo — APPROVED** [review]
Updated the PR Title and Description a bit since the changes are more of a refactor. See if it makes sense.

**Comment by waynemo** [comment]
> looks Good Pavel.
> 
> Please wait for other team member reviews to see if we want to define all color in one file and want to have selectors file independent of color.
> 
> Once, this is merged Please also send PR for teams_interop_merge as we added additional radio box and texts for join call screen.


@iaulakh we should finish the teams_interop_merge->feature/teams_interop merge first. Then we can have a main->feature/teams_interop merge.

**Comment by waynemo** [comment]
@pavelprystinka, looks like there's some intermediate commits, so let's clean up some of those messages when we squash merge.

**Pull request merged**
